src/hooks/useHandleRequestError.js

```diff
@@ -0,0 +1,31 @@
+import { useIntl } from 'react-intl';
```

Comment:
>I feel like I have been told not to import react libraries into files that aren't react components. Did the linter yell about this?

No, because this isn't a vanilla utility function, it is a custom hook.

Reply:
I'm not sure if it makes more sense as a custom hook or a utility function, so I just left it as a hook for now.